.github/workflows/ci.yml

```diff
@@ -14,15 +14,15 @@ jobs:
 
     strategy:
       matrix:
-        os: [macOS-latest]
```

Comment:
Are these intended for this PR?

Reply:
Ah, yes. Forgot to pull those out. Removed this commit from this branch